packages/@storefront/core/src/core/url-beautifier/details/parser.ts

```diff
@@ -62,6 +69,12 @@ export class DetailsUrlParser extends UrlParser<UrlBeautifier.DetailsUrlState> {
 
     return variants;
   }
+
+  static extractFilteredQueryParams(queryParams: {[key: string]: string}) {
```

Comment:
See above re: whether or not this function is actually necessary.

Alternatively, if you decide that it is necessary, it should have 1 or more accompanying tests.

Reply:
Removed unnecessary function.